core/src/wsv.rs

```diff
@@ -694,6 +699,15 @@ impl WorldStateView {
         Ok(f(account))
     }
 
+    fn account(&self, id: &AccountId) -> Result<&Account, FindError> {
+        self.domain(&id.domain_id).and_then(|domain| {
+            domain
+                .accounts
+                .get(id)
+                .ok_or_else(|| FindError::Account(id.clone()))
+        })
+    }
```

Comment:
> What is the utility of these private getters?

none in particular, but we have it in other places. If you want this change it would require a systematic audit of wsv. I don't think we have the bandwidth for that atm